docs/contribution_guide.md

```diff
@@ -77,3 +77,20 @@ https://captum.ai/tutorials/House_Prices_Regression_Interpret
 **Multimodal**
 - You can use VQA model and dataset described here:
 https://captum.ai/tutorials/Multimodal_VQA_Captum_Insights
+
+
+## Docstring style
+
+Docstring is required for all public APIs to provide users the details of the arguments and returns. [Our API documentation](https://captum.ai/api/) is generated from the docstring. Captum adopts a customized docstring format modified on top of [Google style](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html). Specifically, each argument should be listed as `arg_name (type): description` in the `Args:` section. The argument typing convention:
+- primitive types: `int`, `str`, `float`, `bool`
+- common collection types: `list`, `tuple`, `dict`
+    - [PEP 585](https://peps.python.org/pep-0585/#implementation) has deprecated the duplicate types: `List`, `Tuple`, `Dict`
+    - element types: `list[int]`, `dict[int, str]`
+- other foundamental types: `Any`, `Callable`, `Interable`
```

Comment:
The full set of capitalized types should be: "`Callable`", "`Any`", "`Iterable`", "`Iterator`", "`Generator`".

Reply:
Sure, these are just examples. Python surely has more built-in types. No need to list everything.

Reply:
@aobo-y Yeah, there are a more types and would be rather tedious to list them. Those were the types that I found when I inspected the docstrings in Captum's code, and I thought it would be easier for users as they were seemingly common. I do like your idea though of combining a bunch of the types into a single "`Interable`"!